cli/cmd/outputs.go

```diff
@@ -64,3 +66,46 @@ func (c *cliState) FormatJSONString(s string) (string, error) {
 	}
 	return string(pretty), nil
 }
+
+// Used to clean CSV inputs prior to rendering
+func csvCleanData(input []string) []string {
+	var data []string
+	for _, h := range input {
+		data = append(data, strings.Replace(h, "\n", "", -1))
+	}
+	return data
+}
+
+// Used to produce CSV output
+func renderAsCSV(headers []string, data [][]string) (error, string) {
```

Comment:
As a standard we always return the `error` as the last value of a function:
```suggestion
func renderAsCSV(headers []string, data [][]string) (string, error) {
```
For more info look at https://golang.org/doc/tutorial/handle-errors